openslides/assignments/projector.py

```diff
@@ -20,6 +20,11 @@ def get_context(self):
             # Detail slide.
             if not Assignment.objects.filter(pk=pk).exists():
                 raise ProjectorException('Election does not exist.')
+        poll_id = self.config_entry.get('poll')
```

Comment:
Indent this line and the following because they are only required for detail slides.